### PR TITLE
Mockup

### PR DESCRIPTION
**Pull request by sergiogval**
On this request it's merged into the main branch the setup for the project as well as the first part for the mobile version. In complies with all requeriments for the task.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>VALERA Inc.</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/mobile.css">
+</head>
+
+<body>
+  <header>
```
Comment (rustam-tolipov) [inline index.html:14]: - [x] You should have some margin in header tag. I kindly suggest fixing this issue.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <nav>
+      <i class="fa fa-bars"></i>            
+    </nav>
+  </header>
+  <section>
```
Comment (rustam-tolipov) [inline index.html:19]: - [x] You should have some margin in this section tag. Don't forget to add here as well.

File: README.md [hunk 3]
```diff
@@ -1,59 +1,40 @@
...
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+A PC with a Windows 10 or Linux Ubuntu operating system. It has to have GIT installed.
 
 ### Setup
+Go to the following site and clone the repository (https://github.com/sergiogval/sergiogval.github.io)
+## Author
```
Comment (rustam-tolipov) [inline README.md:31]: - [ ] [OPTIONAL] Hey that would be better if you add your real name here to make your perfect readme BRILLILIANT 💎.

File: index.html (hunk 2 above)
Comment (rustam-tolipov) [inline index.html:19]: - [x] Your content should be aligned with flexbox. I suggest using flexbox to add some gap between tags 🍀

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
### Hi @sergiogval  ✋ 
Good job so far in building the project 🥇 , I can see how much effort you made in order to achieve this point! 😊 

However, there are some issues that you still need to work on to go to the next project but you are almost there! 💅

Your linters set up correctly ✔️ 
You followed the GitHub flow ✔️ 

## Required Changes ♻️
 _Check the comments under the review._

 ## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._** 

 As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sergiogval) [reply index.html:19]: done, every element on the body has a margin.

**Review by mmsesay — APPROVED** [review]
Hi @sergiogval , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mmsesay** [comment]
- [x] [OPTIONAL] It seems you miss-spell the font name at lines 48 and 61 of your CSS file. And the spelling mistake makes your headline fonts different from the design. Please try to correct the font spelling.

**Comment by sergiogval** [comment]
Merging the branches with the CR approval.

**Pull request merged**